### ISSUE TITLE
ch05 distributed image is incorrect?

### ISSUE THREAD

**Issue by morefreeze** [progit/progit2#357]
It appears in [here](https://github.com/progit/progit2/blob/master/book/05-distributed-git/sections/contributing.asc). From context (start with line 120), we know Jessica works on both branch featureA and featureB, John works on featureA only, and Josie works on featureB (featureBee). So let's take a look at the [image](https://github.com/progit/progit2/blob/master/book/05-distributed-git/images/managed-team-flow.png) about workflow. John and Josie their name draw conversely in square box, is it right?

So I recommend whether we rename John as Alice who works on featureA, Josie as Bob who works on featureB(ee), Jessica as Candy who works on both. The origin naming is too confused.

**Comment by jnavila** [comment]
@ben Would you prefer changing the text or the picture (picture would make more sense)?

**Comment by ben** [comment]
Yeah, changing the figure makes more sense, though I like the suggestion to change the names throughout. Can we use [Carol](https://en.wikipedia.org/wiki/Alice_and_Bob) instead of Candy, though?

**Comment by jnavila** [comment]
Sorry, I don't understand your question. Where does Candy appear in this story?

**Comment by jnavila** [comment]
Ah, OK, @ben. You meant changing the text as described by @morefreeze . Well, I don't have the editor application for the pictures, so this change can only go half-way.

**Comment by YueLinHo** [comment]
So, using application `Sketch` on Mac to edit `\diagram-source\progit.sketch` file, then exporting png file.
 - Josie -> **Alice** (work on feature **A**)
 - John -> **Bob** (work on feature **B**)
 - Jessica -> **Carol** (work on feature **A** and **B**)

And modifying the file `\book\05-distributed-git\sections\contributing.asc`
 - John -> **Alice** (work on feature **A**)
 - Josie -> **Bob** (work on feature **B**)
 - Jessica -> **Carol** (work on feature **A** and **B**)

**Comment by YueLinHo** [comment]
Why close it?

**Comment by morefreeze** [comment]
@YueLinHo Sorry, I thought you have edit the picture and merge PR. It's my mistake.

**Comment by YueLinHo** [comment]
:smile: I wish I could. I need a Mac and Sketch 3.

**Comment by YueLinHo** [comment]
@schacon said:

> Jessica is my wife's name and John was her father's name.

cf. https://github.com/progit/progit2/pull/1006#issuecomment-368390960

**Comment by YueLinHo** [comment]
Actually, I have created a new SVG file for this figure. And I can rename now. Here is the png exporting file from SVG file:
![image](https://user-images.githubusercontent.com/5831804/52617971-76a74c80-2ed8-11e9-82ba-7ceb9eaac690.png)


But, I have few questions:
1. Is rename good? or just close this issue
2. If we go renaming, I want to know the actual font face. (Consolas is used for now.)

**Comment by YueLinHo** [comment]
Rename Candy to Carol (cf. [ben's comment](https://github.com/progit/progit2/issues/357#issuecomment-151225148))
![image](https://user-images.githubusercontent.com/5831804/52619204-0a2e4c80-2edc-11e9-9dda-baecc49d4321.png)

**Comment by morefreeze** [comment]
I absolutely perfer new image but also respect Scott original naming.

1. New naming is more clear for readers
1. Scott is familiar with original naming without doubt

**Comment by KZiemian** [comment]
The true standing block seems to be creating new graphics in the same graphics style as Mac's Sketch 3 (I don't know nothing about Macs so I maybe write stupid things). If @YueLinHo work can be utilized I will try (and probably fail) to make PR with appropriate changes. But, first I need to learn more Asciidoctor and this is can be a big stumbling block.
